helix-view/src/document.rs

```diff
@@ -1210,6 +1218,12 @@ impl Document {
                     .ensure_invariants(self.text.slice(..));
             }
 
+            for view_data in self.view_data.values_mut() {
+                view_data.view_position.anchor = transaction
+                    .changes()
+                    .map_pos(view_data.view_position.anchor, Assoc::Before);
+            }
```

Comment:
@pascalkuthe this is the only remaining direct access of view_position outside of the access methods, but I think we should keep it because:

1. Since this involves mutating map entries, I think the alternatives of getting view positions then mutating and calling `set_view_position`, or making a specialised method to map them will be messier.
2. If/when we move other parts of `View` in to `ViewData` (`jumps`, `object_selections`) it would make sense to mutate them together in this for loop.
3. This is internal to `mod document`

Reply:
Yeah that's fine